### PR TITLE
[FLINK-30357][doc] Wrong link in connector/jdbc doc.

### PR DESCRIPTION
**Pull request by GOODBOY008**
Link https://issues.apache.org/jira/browse/FLINK-30357.

### CODE REVIEW

**Comment by GOODBOY008** [comment]
@leonardBang Please code review. If you are available. Also I had open a [pr](https://github.com/apache/flink/pull/21482) for branch `release-1.16`.

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
@GOODBOY008 Please also check the Chinese translation

File: docs/content.zh/docs/connectors/table/jdbc.md [hunk 1]
```diff
@@ -672,12 +672,6 @@ Flink 支持连接到多个使用方言（dialect）的数据库，如 MySQL、O
       <td></td>
       <td><code>DECIMAL(20, 0)</code></td>
     </tr>
-    <tr>
-      <td><code>BIGINT</code></td>
-      <td></td>
-      <td><code>BIGINT</code></td>
-      <td><code>BIGINT</code></td>
-    </tr>
```
Comment (MartijnVisser) [inline docs/content.zh/docs/connectors/table/jdbc.md:680]: Why is this removed?

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
@GOODBOY008 Please check the removed content from the docs

File: docs/content.zh/docs/connectors/table/jdbc.md (hunk 1 above)
Comment (GOODBOY008) [reply docs/content.zh/docs/connectors/table/jdbc.md:680]: I think it's dumplicated.

Comment (MartijnVisser) [reply docs/content.zh/docs/connectors/table/jdbc.md:680]: I don't think it's duplicated, because there's a BIGINT INT UNSIGNED / BIGINT UNSIGNED data type for MySQL (which are the first two lines), but there's also a regular `BIGINT` data type which exists for Postgres.

Comment (GOODBOY008) [reply docs/content.zh/docs/connectors/table/jdbc.md:680]: I get it,Thanks for your patient reply.

**Review by MartijnVisser — APPROVED** [review]
LGTM
